test/Microsoft.TestPlatform.Common.UnitTests/Filtering/FastFilterTests.cs

```diff
@@ -321,13 +313,74 @@ public void FastFilterShouldThrowExceptionForUnsupportedOperatorOperationCombina
             ImmutableHashSet<string>.Builder filterHashSetBuilder = ImmutableHashSet.CreateBuilder<string>();
             try
             {
-                var filter = new FastFilter("dummyName", filterHashSetBuilder.ToImmutableHashSet(), Operation.Equal, Operator.And);
+                var filter = new FastFilter(ImmutableDictionary.CreateRange(new[] { new KeyValuePair<string, ISet<string>>("dummyName", filterHashSetBuilder.ToImmutableHashSet()) }), Operation.Equal, Operator.And);
             }
             catch (Exception ex)
             {
                 Assert.IsTrue(ex is ArgumentException);
                 Assert.AreEqual("An error occured while creating Fast filter.", ex.Message);
             }
         }
+
+        [TestMethod]
+        public void MultiplePropertyNames()
```

Comment:
Could you also add a test for the other pattern? e.g. "FQN!=Test1&TestCategory!=SkipWhenLiveUnitTesting"

Reply:
Will do?